test/index.spec.js

```diff
@@ -110,7 +110,7 @@ describe('CID', () => {
       expect(cid).to.have.property('version', 1)
       expect(cid).to.have.property('multihash')
 
-      expect(cid.toBaseEncodedString()).to.be.eql(cidStr)
+      expect(cid.toBaseEncodedString('base58btc')).to.be.eql(cidStr)
```

Comment:
Please change the `expect` to use no base parameter, else the test doesn't match its description and is the same test as the one above.